tools/vex2pdf.json

```diff
@@ -0,0 +1,51 @@
+{
+  "tool": {
+    "name": "vex2pdf",
+    "publisher": "Salem B.",
+    "description": "A Rust command-line tool and library to convert CycloneDX (VEX/VDR/SBOM) JSON or XML documents to PDF reports with support for vulnerability analysis visualization.",
+    "repository_url": "https://gitlab.com/jurassicLizard/vex2pdf",
+    "website_url": "https://crates.io/crates/vex2pdf",
+    "capabilities": [
+      "SBOM",
+      "VDR/VEX"
+    ],
+    "availability": [
+      "OPEN_SOURCE"
+    ],
+    "functions": [
+      "TRANSFORM"
+    ],
+    "analysis": [
+      "RESOURCE_REPORTING",
+      "SECURITY_VULNERABILITIES"
+    ],
+    "transform": [
+      "BOM_SERIALIZATION_FORMAT"
```

Comment:
the description of the tool transforms from JSOM/XML to PDF.
this behavior doesn't cover "BOM_SERIALIZATION_FORMAT"
> Transforms the BOM between supported serialization formats such as XML and JSON.

could you elaborate on this?